api/client/service/update.go

```diff
@@ -30,8 +31,8 @@ func newUpdateCommand(dockerCli *client.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.String("image", "", "Service image tag")
-	flags.StringSlice("command", []string{}, "Service command")
-	flags.StringSlice("arg", []string{}, "Service command args")
+	flags.String("command", "", "Service command")
+	flags.String("args", "", "Service command args")
```

Comment:
Yes, it's going to be pretty annoying if you need to quote things in the command. It's not a perfect solution, but it feels like the best option right now. 

> We could consider treating extra arguments that aren't associated with a flag as a replacement command/args, but this might be very unintuitive for other reasons

Ya, for one it's still going to be inconsistent with `service create`, because the first positional arg to `create` is the image not the command. I'm pretty sure we don't want to force the image as a positional in `update`.